cluster/addons/registry/registry.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta2
+kind: Deployment
+metadata:
+  name: registry
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: registry
+    spec:
+      containers:
+      - name: registry
+        image: registry
```

Comment:
the rs definition in https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/registry/README.md pointed at registry:2 here. why does this use a different version?

Reply:
In this doc https://hub.docker.com/r/library/registry/tags/.
I think it will pull latest version